core/block.ts

```diff
@@ -558,9 +558,10 @@ export class Block implements IASTNodeLocation, IDeletable {
   /**
    * Bump unconnected blocks out of alignment.  Two blocks which aren't actually
    * connected should not coincidentally line up on screen.
+   *
+   * @deprecated v11 - You should not need to call bump neighbours directly.
```

Comment:
This should also be removed then I assume? I'm not sure why this method even actually exists on non-rendered blocks in the first place, but... if we were going to deprecate this one it would need a different message referencing the fact that bumping doesn't make sense in non-rendered workspaces

Reply:
It has to exist for type polymorphism reasons.